src/components/category/catGroup.js

```diff
@@ -1,13 +1,17 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+
 import CatItem from './catItem';
 
 
 const catGroup = (props) => {
   let items = props.items.map(item => {
+    if(!item.name) return false;
     return (
       <CatItem 
         key={item.id}
-        name={item.name} />
+        name={item.name}
+        clicked={() => props.itemClicked(props.idCat, item.name)} />
```

Comment:
just checking, props.idCat is not the same as item.id? i'm not sure, just checking

Reply:
They're different, the **props.idCat** is the parent's id (dtrink type, alcohol) and the **item.id** is the id of every children inside that category.